arcade/drawing_support.py

```diff
@@ -97,3 +97,37 @@ def make_transparent_color(color: Color, transparency: float):
     :param float transparency: Transparency
     """
     return color[0], color[1], color[2], transparency
+
+
+def uint24_to_three_byte_color(color: int) -> RGB:
+    """
+    Given an int between 0 and 16777215, return a RGB color tuple.
+
+    :param int color: 3 byte int
+    """
+    return [(color & (255 << i)) >> i for i in (16, 8, 0)]
+
+
+def uint32_to_four_byte_color(color: int) -> RGBA:
+    """
+    Given an int between 0 and 4294967295, return a RGBA color tuple.
+
+    :param int color: 4 byte int
+    """
+    return [(color & (255 << i)) >> i for i in (24, 16, 8, 0)]
```

Comment:
Returning a  tuple is faster than returning a list when all elements are immutable